radio/src/pulses/pulses.cpp

```diff
@@ -523,7 +519,6 @@ bool setupPulsesInternalModule(uint8_t protocol)
     case PROTOCOL_CHANNELS_PXX1_PULSES:
       intmodulePulsesData.pxx.setupFrame(INTERNAL_MODULE);
 #if defined(INTMODULE_HEARTBEAT)
-      mixerSchedulerResetTimer();
       mixerSchedulerSetPeriod(INTERNAL_MODULE, INTMODULE_PXX1_SERIAL_PERIOD + 1000 /* backup */);
```

Comment:
@3djc I think that modules / protos using fixed period (with or without heartbeat) do not need to set the period every time, as it does not change, and even if that would change, the new period is used only if the timer hits (set only in `mixerSchedulerStart()` and `MIXER_SCHEDULER_TIMER_IRQHandler()`). For PXX2, this is a different story, as I'm not sure the module is reset when going from TOOLS_PERIOD to the "normal" period.